projects/kit/components/input-date-range/input-date-range.component.ts

```diff
@@ -134,7 +135,10 @@ export class TuiInputDateRangeComponent
     open = false;
 
     readonly maxLengthMapper: TuiMapper<TuiDay, TuiDay> = MAX_DAY_RANGE_LENGTH_MAPPER;
-    readonly dateFiller$ = this.dateTexts$.pipe(pluck(this.dateFormat));
+    readonly dateFiller$ = this.dateTexts$.pipe(
+        pluck(this.dateFormat),
```

Comment:
`pluck` is deprecated. If we introducing a `map` anyway, let's add extracting in it as well and drop `pluck`.

Reply:
fixed